staging/src/github.com/kubeedge/mapper-framework/README.md

```diff
@@ -43,5 +45,3 @@ mapper
 └── Makefile
 ```
 
-# Where does it come from?
-mapper-framework is synced from https://github.com/kubeedge/kubeedge/tree/master/staging/src/github.com/kubeedge/mapper-framework. Code changes are made in that location, merged into kubeedge and later synced here.
```

Comment:
Should not delete the `Where does it come from?` part, which will be used in the new sub-project.

Reply:
Done.